test/testSanity.js

```diff
@@ -0,0 +1,15 @@
+var expect = require('chai').expect;
```

Comment:
when we enable the linter for tests it will yield for this and a few other things